### PR TITLE
Added github workflow

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-758

- Updated for d-10
- Added github pr workflow

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+on: pull_request
+name: PR Review
+jobs:
+  changelog:
+    name: Changelog should be updated
+    runs-on: ubuntu-20.04
```
Comment (rimi-itk) [inline .github/workflows/pr.yaml]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
+
+      - name: Check that changelog has been updated.
+        run: git diff --exit-code origin/develop -- CHANGELOG.md && exit 1 || exit 0
```
Comment (rimi-itk) [inline .github/workflows/pr.yaml]: `develop` is not always the base of a pull request:

```suggestion
        run: git diff --exit-code origin/${{ github.base_ref }} -- CHANGELOG.md && exit 1 || exit 0
```

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          cd drupal/web/modules/contrib/drupal_psr6_cache
+          # Remove our non-dev dependencies to prevent duplicated Drupal installation
+          # PHP Fatal error:  Cannot redeclare drupal_get_filename() (previously declared in /home/runner/work/drupal_psr6_cache/drupal_psr6_cache/drupal/web/modules/contrib/drupal_psr6_cache/vendor/drupal/core/includes/bootstrap.inc:190) in /home/runner/work/drupal_psr6_cache/drupal_psr6_cache/drupal/web/core/includes/bootstrap.inc on line 190
+          # Use sed to remove the "require" property in composer.json
+          sed -i '/^\s*"require":/,/^\s*}/d' composer.json
+          composer --no-interaction install
+          composer code-analysis
+
+  markdown-coding-standards:
+    runs-on: ubuntu-20.04
```
Comment (rimi-itk) [inline .github/workflows/pr.yaml]:
```suggestion
    runs-on: ubuntu-latest
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.1.0]
+
+- Updated for d-10
```
Comment (rimi-itk) [inline CHANGELOG.md]:
```suggestion
- Updated for Drupal 10
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Added github pr workflow
```
Comment (rimi-itk) [inline CHANGELOG.md]:
```suggestion
- Added GitHub pull request workflow
```

File: phpcs.xml.dist [hunk 6]
```diff
@@ -3,15 +3,17 @@
...
 
   <!-- Show progress of the run -->
   <arg value="p"/>
 
   <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,yml"/>
-  <config name="drupal_core_version" value="8"/>
+  <config name="drupal_core_version" value="9"/>
 
   <rule ref="Drupal">
+    <!-- <exclude name="Drupal.Files.TxtFileLineLength.TooLong"/> -->
```
Comment (rimi-itk) [inline phpcs.xml.dist]: Why is this comment added? It has no effect on the coding standards check (it's a comment).

File: drupal_psr6_cache.info.yml [hunk 7]
```diff
@@ -2,4 +2,4 @@ name: 'PSR-6 Cache'
 type: module
 description: 'PSR-6: Caching Interface for Drupal 8+'
 package: 'Custom'
-core_version_requirement: ^8 | ^9
+core_version_requirement: ^8 | ^9 | ^10
```
Comment (rimi-itk) [inline drupal_psr6_cache.info.yml]:
```suggestion
core_version_requirement: ^9 | ^10
```

or maybe even

```suggestion
core_version_requirement: ^10
```

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: phpcs.xml.dist (hunk 6 above)
Comment (martinyde) [reply phpcs.xml.dist]: fixed

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**